scripts/cibuild

```diff
@@ -31,8 +31,13 @@ if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
             -f docker-compose.ci.yml \
             build activator-aviris-l2
 
+        # COG Clip module tests
         ./scripts/test
```

Comment:
I'd consider reordering things like this:

- Run `update`
- Run `test`
- Build assembly JAR
- Build all container images

This allows us to only have one `docker-compose build` invocation and I think makes the flow logical.

Reply:
@rbreslow done!